apps/ff-site/src/content/events/fil-paris.md

```diff
@@ -1,17 +1,327 @@
 ---
-title: FIL Paris 2023
-created-on: 2023-06-27T15:38:07.082Z
-updated-on: 2023-06-27T15:38:07.098Z
-published-on: 2023-06-27T15:38:07.109Z
+title: FIL Paris
+description: FIL Paris 2023 was a week-long event (July 15–22) in Paris,
```

Comment:
@YuliiaFilecoin - I don't think we should talk about this event as if it happened in the past..

Reply:
I'll change the verbs to present tense!